content/en/blog/_posts/2020-10-12-steering-committee-results.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: blog
+title: "Announcing the 2020 Steering Committee Election Results"
+date: 2020-10-12
+slug: steering-committee-results-2020
+---
+
+**Author**: Kaslin Fields
+
+The [2020 Steering Committee Election](https://github.com/kubernetes/community/tree/master/events/elections/2020) is now complete. In 2019, the committee arrived at its final allocation of 7 seats, 3 of which were up for election in 2020. Incoming committee members serve a term of 2 years, and all members are elected by the Kubernetes Community.
+
+This community body is significant since it oversees the governance of the entire Kubernetes project. With that great power comes great responsibility. You can learn more about the steering committee’s role in their [charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+
+## Results
+
+Congratulations to the elected committee members whose two year terms begin immediately (listed in alphabetical order by GitHub handle):
+
+* **Davanum Srinivas ([@dims](https://github.com/dims)), VMware**
+* **Jordan Liggitt ([@liggitt](https://github.com/liggitt)), Google**
+* **Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables)), Google**
+
+They join existing members Christoph Blecker ([@cblecker](https://github.com/cblecker)), Red Hat; Derek Carr ([@derekwaynecarr](https://github.com/derekwaynecarr)), Red Hat; Nikhita Raghunath ([@nikhita](https://github.com/nikhita)), VMware; and Paris Pittman ([@parispittman](https://github.com/parispittman)), Apple.
```

Comment:
Dims is a returning, re-elected member of the SC.  Should this text be changed to reflect that, or is it fine the way it is?

Reply:
My gut feeling is yes, to show how many people were elected.

Reply:
I was also wondering how best to reflect that. I considered putting (returning) at the end of Dims' line there. Maybe that would work well?
We could add this to the end of line 22:
Davanum Srinivas is returning for his second term on the committee.

Reply:
can we swap "continuing" for "existing"?  Seems clearer, since everyone exists.

Reply:
Added a commit to address this. Swapped "existing" with "continuing" and added the line about Dims returning.